holoviews/element/path.py

```diff
@@ -98,21 +109,31 @@ class Contours(Path):
     level = param.Number(default=None, doc="""
         Optional level associated with the set of Contours.""")
 
-    vdims = param.List(default=[Dimension('Level')], doc="""
+    vdims = param.List(default=[], doc="""
         Contours optionally accept a value dimension, corresponding
-        to the supplied values.""", bounds=(1,1))
+        to the supplied values.""")
 
     group = param.String(default='Contours', constant=True)
 
+    _level_vdim = Dimension('Level') # For backward compatibility
```

Comment:
Ok, I assume this is being injected back into vdims for backwards compatibility as needed. You can ignore the comment immediately above.